e2e/testdata.go

```diff
@@ -37,6 +45,12 @@ func getSamplePod(name, pvcName string) *k8sv1.Pod {
 							ContainerPort: 80,
 						},
 					},
+					VolumeMounts: []k8sv1.VolumeMount{
```

Comment:
This will only work for VolumeMode == Filesystem. For Block, it needs to be:

```
      volumeDevices:
        - name: data
          devicePath: /dev/xvda
```
See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#pod-specification-adding-raw-block-device-path-in-container